src/lexer/src_string.sig

```diff
@@ -1,7 +1,9 @@
 signature SRC_STRING =
 sig
     type yypos = int
-
+    exception StringNotClosed of int * int (* line position * line number *)
```

Comment:
I don't understand the comment on this line? Also may I recommend the name `NotClosed`, since we are in the `SrcString` structure?